spec/features/user_creates_new_book_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+feature 'User creates a new book' do
+  scenario 'successfully' do
+    visit root_path
+
+    create_book("The Lord of the Rings: The Fellowship of the Ring", "J. R. R. Tolkien" )
```

Comment:
Hi Paul, I visited the houndci website and I think you have to pay unless your project is open source? Is houndci automatically run on all thoughbot repos? If so, I can merge in this pull request and just fork/recreate the repo in thoughtbot.